Lib/glyphsLib/glyphdata.py

```diff
@@ -80,6 +80,15 @@ def _get_category(name, unistr, data=glyphdata_generated):
         return ("Symbol", "Geometry")
     if basename.startswith("uniF9"):
         return ("Letter", "Compatibility")
+
+    glyph_info = glyphdata_generated.DEFAULT_GLYPH_DICT.get(name)
+    if glyph_info is None:
+        glyph_info = glyphdata_generated.DEFAULT_GLYPH_DICT.get(basename)
+    if glyph_info is not None:
+        category, subCategory = glyph_info.get('category'), glyph_info.get('subCategory')
+        if (category, subCategory) != (None, None):
+            return (category, subCategory)
```

Comment:
this is confusing. `data` already defaults to `glyphdata_generated`; if one gives a `data` value, then this function should treat that as a replacement for the default value.. But here you still looking up the original `glyphdata_generated` nevertheless.